product-properties-api-v1/spec-product-properties-v1.yaml

```
@@ -0,0 +1,293 @@
+openapi: 3.0.4
+info:
+  title: crowdproj-product-properties
+  description: Microservice for management and provisioning of the physical properties of a product, article, cargo, item
+  contact:
+    name: Sergey Okatov
+    email: sokatov@gmail.com
+  license:
+    name: Apache 2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
+  version: 1.0.0
+servers:
+  - url: http://localhost:8080/v1
+tags:
+  - name: product-property
+    description: property of a product, article, cargo, item
+
+paths:
+  /product-property/create:
```

```suggestion
  /product/property/create:
```
Лучше как путь сделать. Есть еще несколько сервисов для обслуживания продукта